### PR TITLE
Add profile data parser for image retriever models

### PR DESCRIPTION
**Pull request by nv-hwoo**
Added
- new profile data parser for image retriever models
- new tests for the profile data parser
- refactor & clean up `test_llm_profile_data_parser.py`

Example
image

### CODE REVIEW

**Comment by nv-hwoo** [comment]
@fpetrini15 do you know why the CI trigger failed here?

**Comment by fpetrini15** [comment]
Seemed to be a network error. Worked on retry.

**Comment by nv-hwoo** [comment]
Ah I see. Thanks @fpetrini15 👍 Didn't realize there was a respin button :D

File: genai-perf/genai_perf/metrics/image_retrieval_metrics.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        self._base_names["image_throughputs"] = "image_throughput"
+        self._base_names["image_latencies"] = "image_latency"
+
+    @property
+    def request_metrics(self) -> List[MetricMetadata]:
+        base_metrics = super().request_metrics  # base metrics
+        return base_metrics + self.IMAGE_RETRIEVAL_REQUEST_METRICS
+
+    @property
+    def system_metrics(self) -> List[MetricMetadata]:
```
Comment (debermudez) [inline genai-perf/genai_perf/metrics/image_retrieval_metrics.py:63]: do we need this at all?
wont the parent class's version be called if this isn't defined?

File: genai-perf/tests/test_image_retrieval_profile_data_parser.py
Comment (debermudez) [inline genai-perf/tests/test_image_retrieval_profile_data_parser.py:1]: Lots of changes on this file. 
Can you give me the high level reason for these changes?

File: genai-perf/genai_perf/metrics/image_retrieval_metrics.py (hunk 1 above)
Comment (nv-hwoo) [reply genai-perf/genai_perf/metrics/image_retrieval_metrics.py:63]: good catch! yeah I don't think we need this.

File: genai-perf/tests/test_image_retrieval_profile_data_parser.py
Comment (debermudez) [reply genai-perf/tests/test_image_retrieval_profile_data_parser.py:1]: perfect. that was exactly what i needed to understand the context.
and yes....i tagged wrong file 😮‍💨 . thanks for understanding anyway.

**Pull request merged**